src/NuGet.Clients/PackageManagement.UI/Models/LoadingStatusViewModel.cs

```diff
@@ -68,7 +68,7 @@ public bool IsMultiSource
             nameof(IsMultiSource),
             typeof(bool),
             typeof(LoadingStatusViewModel),
-            new PropertyMetadata(true, OnIsMultiSourcePropertyChanged));
+            new PropertyMetadata(false, OnIsMultiSourcePropertyChanged));
```

Comment:
Changing default of IsMultiSource to false. Ideally it should be tri-state boolean with null being default. But that would make this fix too noisy.